tests/vcs/git/GitCommitBearTest.py

```diff
@@ -139,6 +139,17 @@ def test_shortlog_checks_shortlog_trailing_period(self):
         self.assertEqual(self.run_uut(shortlog_trailing_period=False), [])
         self.assertEqual(self.run_uut(shortlog_trailing_period=None), [])
 
+    def test_shortlog_checks_imperative(self):
+        self.git_commit("tag: Add shortlog in imperative")
+        self.assertNotIn("Shortlog of HEAD commit isn't imperative mood",
+                         ''.join(self.run_uut()))
+        self.git_commit("Added invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
+        self.git_commit("Adding another invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
```

Comment:
Can't we fid the word which is a "verb" ?
Because "maximizing function" is the object in that sentence right (and "maximizing" is adjective? (not sure)) ? So we should be checking only the word which is POS tagged as verb